akka-stream/src/main/scala/akka/stream/impl/ActorFlowMaterializerImpl.scala

```diff
@@ -222,6 +224,13 @@ private[akka] object StreamSupervisor {
   def props(settings: ActorFlowMaterializerSettings): Props = Props(new StreamSupervisor(settings))
 
   final case class Materialize(props: Props, name: String) extends DeadLetterSuppression
+
+  /** Testing purpose */
+  final case object GetChildren
```

Comment:
So the thing is, I believe we can make this a proper API. We need to discuss the details, I have some ideas. Good enough for this PR though, just wanted to mention.